docs/whatsnew/2.1.rst

```diff
@@ -0,0 +1,215 @@
+
+    >>> hek_results, jsoc_results = results['hek'], results['jsoc']
+
+In the first example you can see the search results from HEK are not too useful by default.
+The results are truncated as the long urls for the thumbnail take up too many characters and HEK can return up to 100 columns of information.
+With a new method ``show()`` on the results itself, you directly select the columns to display::
+
+    >>> hek_results.show('event_peaktime', 'obs_instrument', 'fl_peakflux')
+    event_peaktime   obs_instrument fl_peakflux
+    ------------------- -------------- -----------
+    2010-08-01T03:40:37            AIA     1027.64
+    2010-08-01T03:40:44            AIA     1441.78
+
+To find the keys for all of the columns you can do::
+
+    >>> hek_results.keys()
+    ['gs_thumburl',
+    'comment_count',
+    'hpc_bbox',
+    ...
+    'area_unit',
+    'obs_lastprocessingdate',
+    'refs']
+
+In addition, if you pass the entire search query into ``Fido.fetch()`` and it will ignore results that have no corresponding data files to retrieve.
+
+.. _whatsnew-2.1-refactor:
+
+Fido Results Refactor
+=====================
+
+While working on the above change, several changes were made to the way that results are returned from ``Fido.search()`` and the search methods of the underlying clients.
+
+We have tried to minimize any breaking changes here and we believe that for most users the difference between 2.0 and 2.1 will be minor.
+
+The key highlights you will want to be aware of are:
+
+* Previously slicing the result of ``Fido.search()`` (a `~sunpy.net.fido_factory.UnifiedResponse` object) so that it had a length of one returned another `~sunpy.net.fido_factory.UnifiedResponse` object, now it will return a `~sunpy.net.base_client.QueryResponseTable` object, which is a subclass of `astropy.table.Table`.
+
+* All result objects contained within the results of a ``Fido.search()`` are now `~sunpy.net.base_client.QueryResponseTable` objects (or subclasses thereof).
+  These objects are subclasses of `astropy.table.Table` and can therefore be filtered and inspected as tabular objects, and the modified tables can be passed to ``Fido.fetch``.
+
+* The keys available to be used when formatting the ``path=`` argument to ``Fido.fetch()`` have changed. This is to standardise them over the results from more clients and make them easier to use.
+  You can use the `~.UnifiedResponse.path_format_keys` method to see all the possible keys for a particular search.
+
+* The search results object returned from ``Fido.search`` now correctly counts all results in it's `~sunpy.net.fido_factory.UnifiedResponse.file_num` property.
+
+* Results from the `~sunpy.net.dataretriever.NOAAIndicesClient` and the `~sunpy.net.dataretriever.NOAAPredictClient` no longer have ``Start Time`` or ``End Time`` in their results table as the results returned from the client are not dependant upon the time parameter of a search.
+
+.. _whatsnew-2.1-contours:
+
+Finding map contours
```

Comment:
I think this entry should be pushed down to between the reprojection entry and the performance entry